examples/resnet/conda.yaml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - python=3.8.13
   - pip=20.2
   - pip:
+      - azureml-fsspec
```

Comment:
nit: this is probably not required? Since it is on the aml side there is no use for fsspec.

Reply:
Oh, sry. We may need that still. As in aml system, we call `fsspec.info` to check if the input is file or folder.

Reply:
Doesn't this change to local file/folder when seen from the aml_pass_runner or aml_evaluation_runner? 

The check is only done by the aml system running on the local environment. but this conda yaml is for the azureml job environment.

Reply:
Ic